e2e-tests/ukef/cypress/e2e/journeys/portal/deal-cancellation/gef/AIN-deal-cancellation-updates-updates-portal.spec.js

```diff
@@ -0,0 +1,120 @@
+import { DEAL_STATUS, PORTAL_ACTIVITY_LABEL, TFM_URL, UKEF } from '@ukef/dtfs2-common';
+import relative from '../../../../relativeURL';
+import applicationActivities from '../../../../../../../gef/cypress/e2e/pages/application-activities';
+import statusBanner from '../../../../../../../gef/cypress/e2e/pages/application-status-banner';
+import MOCK_USERS from '../../../../../../../e2e-fixtures/portal-users.fixture';
+import { MOCK_APPLICATION_AIN_DRAFT } from '../../../../../../../e2e-fixtures/gef/mocks/mock-deals';
+import { anIssuedCashFacility } from '../../../../../../../e2e-fixtures/mock-gef-facilities';
+import { yesterday, today } from '../../../../../../../e2e-fixtures/dateConstants';
+import { PIM_USER_1 } from '../../../../../../../e2e-fixtures';
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+context('GEF AIN deal - When TFM submits a deal cancellation - Portal status and activity feed should be updated', () => {
+  let gefDeal;
+  let dealId;
+
+  const mockFacility = anIssuedCashFacility({ facilityEndDateEnabled: true });
+
+  before(() => {
+    cy.insertOneGefDeal(MOCK_APPLICATION_AIN_DRAFT, BANK1_MAKER1).then((insertedDeal) => {
+      gefDeal = insertedDeal;
+      dealId = gefDeal._id;
+
+      // updates a gef deal to have relevant fields
+      cy.updateGefDeal(dealId, MOCK_APPLICATION_AIN_DRAFT, BANK1_MAKER1);
+
+      cy.createGefFacilities(dealId, [mockFacility], BANK1_MAKER1).then((createdFacilities) => {
+        gefDeal.facilities = createdFacilities;
+        cy.makerLoginSubmitGefDealForReview(gefDeal);
+        cy.checkerLoginSubmitGefDealToUkef(gefDeal);
+        cy.clearSessionCookies();
+
+        cy.forceVisit(TFM_URL);
```

Comment:
should we be using forceVisit?

Reply:
I followed the existing tests - turns out we don't need this. Have removed from all deal cancellation tests in this suite/directory.